src/libs/workspace-utils.ts

```diff
@@ -188,10 +198,10 @@ export interface WorkspaceAccessInfo {
 export const canEditWorkspace = ({
   accessLevel,
   workspace: { isLocked },
-}: WorkspaceAccessInfo): { value: boolean; message?: string } =>
+}: WorkspaceAccessInfo | WorkspaceWrapper): { value: boolean; message?: string } =>
```

Comment:
The intent here is that `WorkspaceAccessInfo` is the subset of `WorkspaceWrapper` that this function actually needs.

```suggestion
}: WorkspaceAccessInfo): { value: boolean; message?: string } =>
```

Reply:
There was an incompatibility between `WorkspaceAccessInfo` and `WorkspaceWrapper` - so I fixed that, and put the type back to just `WorkspaceAccessInfo`